### PR TITLE
tests: make TestFallbackTSOConsistency stable

### LINKED ISSUE #7407: TestFallbackTSOConsistency is unstable
### Which jobs are failing
```
2023-11-21T17:29:24.9392140Z     consistency_test.go:135: 
2023-11-21T17:29:24.9393115Z         	Error Trace:	/home/runner/work/pd/pd/tests/integrations/tso/consistency_test.go:135
2023-11-21T17:29:24.9394798Z         	            				/home/runner/work/pd/pd/tests/integrations/tso/consistency_test.go:197
2023-11-21T17:29:24.9396109Z         	            				/opt/hostedtoolcache/go/1.21.4/x64/src/runtime/asm_amd64.s:1650
2023-11-21T17:29:24.9397051Z         	Error:      	Received unexpected error:
2023-11-21T17:29:24.9398691Z         	            	rpc error: code = Unknown desc = [PD:etcd:ErrEtcdTxnConflict]etcd transaction failed, conflicted and rolled back
2023-11-21T17:29:24.9400337Z         	Test:       	TestMicroserviceTSOConsistency/TestFallbackTSOConsistency
```

### CI link

https://github.com/tikv/pd/actions/runs/6947385528/job/18901051379?pr=7406

### PR DESCRIPTION
**Pull request by lhy1024**
### What problem does this PR solve?

Issue Number: Close #7407

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by JmPotato — COMMENTED** [review]
Why not just use `testutil.Eventually`?

File: tests/integrations/tso/consistency_test.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ import (
 	tso "github.com/tikv/pd/pkg/mcs/tso/server"
 	tsopkg "github.com/tikv/pd/pkg/tso"
 	"github.com/tikv/pd/pkg/utils/tempurl"
-	pd "github.com/tikv/pd/pkg/utils/testutil"
+	tu "github.com/tikv/pd/pkg/utils/testutil"
```
Comment (JmPotato) [inline tests/integrations/tso/consistency_test.go:30]: Why not just use the original package name?

Comment (lhy1024) [reply tests/integrations/tso/consistency_test.go:30]: In most places, it is called `tu`

**Review by rleungx — APPROVED** [review]

**Review by JmPotato — APPROVED** [review]

**Pull request merged**